npm-management.md

```diff
@@ -0,0 +1,41 @@
+# NPM Management
+
+We have a number of modules under the Node.js Foundation including:
+
+* [citgm](https://github.com/nodejs/citgm)
+* [llnode](https://github.com/nodejs/llnode)
+* [node-gyp](https://github.com/nodejs/node-gyp)
+* [node-inspect](https://github.com/nodejs/node-inspect)
+* [node-report](https://github.com/nodejs/node-report)
+
+We need to make sure that we have continuity in terms of being able to publish
+and update these modules.
+
+We have decided to manage these modules as follows:
+
+* Create a user called [`nodejs-foundation`][] who we always add as one of the
+  collaborators with admin rights and for which the password is maintained by
+  the build workgroup.
+* We would then add individuals as collaborators who can also publish.
+  Generally, a module push will be done by the additional collaborators.
+  The `nodejs-foundation` user is intended to be used as a backup as opposed
+  to being part of the regular publishing flow.
+* In the cases where collaborators other than `nodejs-foundation`
+  cease to be active, the build workgroup would provide continuity by using the
+  `node-foundation` user to add additional collaborators who would have the
+  ability to push the module. The `node-foundation` user could also be used to
+  remove collaborators if that was ever necessary.
+* The purpose of the `nodejs-foundation` user is not to enable Build
+  Workgroup members to publish npm modules, that should be left to the
+  module collaborators.
+
```

Comment:
We might also can mention that developers should enable 2FA on npm as well here.

Reply:
@watilde The foundation account does not even enable 2FA on npm (can we fix that?) ...so probably not really worth mentioning that for now

Reply:
I thought it's the same situation with `@nodejs-github-bot` account without `2FA` in GitHub and collaborators need to enable their 2FA. We can mention later after landing this.